app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def index
+    # it'll show all users
+  end
+
+  def show
+    # it'll show users
+  end
```

Comment:
[REQUIRED] Just like your index action kindly add code that finds and retrieves the  user specified in your URL id parameter